recipe/meta.yaml

```
@@ -1,47 +1,59 @@
 {% set name = "twine" %}
-{% set version = "4.0.2" %}
+{% set version = "6.0.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/twine-{{ version }}.tar.gz
-  sha256: 9e102ef5fdd5a20661eb88fad46338806c3bd32cf1db729603fe3697b1bc83c8
+  sha256: 36158b09df5406e1c9c1fb8edb24fc2be387709443e7376689b938531582ee27
 
 build:
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
   number: 0
-  skip: true  # [py<37]
+  skip: true  # [py<38]
   entry_points:
     - twine = twine.__main__:main
 
 requirements:
   host:
     - python
-    - setuptools >=45
+    - pip
     - wheel
+    - setuptools >=61.2
     - setuptools_scm >=6.0
-    - pip
   run:
     - python
     - pkginfo >=1.8.1
     - readme_renderer >=35.0
     - requests >=2.20
     - requests-toolbelt !=0.9.0,>=0.8.0
     - urllib3 >=1.26.0
-    - importlib-metadata >=3.6
-    - keyring >=15.1
+    - importlib-metadata >=3.6  # [py<310]
+    - keyring >=15.1            # [not s390x]
     - rfc3986 >=1.4.0
     - rich >=12.0.0
+    - packaging
+  run_constrained:
+    - keyring >=15.1            # [s390x]
 
 test:
+  requires:
+    - pip
+    - pretend
+    - pytest
+  source_files:
+    - twine
+    - tests
   imports:
     - twine
```

```suggestion
    - twine
    - twine.commands
```
Honestly, probably not necessary, but it's listed [here](https://github.com/pypa/twine/blob/6.0.1/pyproject.toml#L70), so it would be nice if we could add it explicitly.